dotnet-desktop-guide/framework/wpf/controls/listview-overview.md

```diff
@@ -21,7 +21,7 @@ The <xref:System.Windows.Controls.ListView> control provides the infrastructure
 
 ## What Is a ListView?  
 
- The <xref:System.Windows.Controls.ListView> control is an <xref:System.Windows.Controls.ItemsControl> that is derived from <xref:System.Windows.Controls.ListBox>. Typically, its items are members of a data collection and are represented as <xref:System.Windows.Controls.ListViewItem> objects. A <xref:System.Windows.Controls.ListViewItem> is a <xref:System.Windows.Controls.ContentControl> and can contain only a single child element. However, that child element can be any visual element.  
+ The <xref:System.Windows.Controls.ListView> control is an <xref:System.Windows.Controls.ItemsControl> derived from <xref:System.Windows.Controls.ListBox>. Typically, its items are members of a data collection and are represented as <xref:System.Windows.Controls.ListViewItem> objects. A <xref:System.Windows.Controls.ListViewItem> is a <xref:System.Windows.Controls.ContentControl> and can contain only a single child element. However, that child element can be any visual element.  
```

Comment:
I'm not too keen on this change. It feels like you lose the concept that `ListView` derives from `ListBox` and both controls are `ItemsControl` types. Maybe this whole sentence can be restructured or perhaps `ItemsControl` can be omitted. This is the only place `ItemsControl` is referenced, so I don't really think it's important to point that out. What about this?
```suggestion
 The <xref:System.Windows.Controls.ListView> derives from <xref:System.Windows.Controls.ListBox>. Typically, its items are members of a data collection and are represented as <xref:System.Windows.Controls.ListViewItem> objects. A <xref:System.Windows.Controls.ListViewItem> is a <xref:System.Windows.Controls.ContentControl> and can contain only a single child element. However, that child element can be any visual element.  
```

Reply:
Your suggestion sounds great. I think it is more eloquent and if it is technically correct then it is a winner.